mixer/pkg/api/grpcServer.go

```diff
@@ -153,7 +153,7 @@ func (s *grpcServer) Check(legacyCtx legacyContext.Context, req *mixerpb.CheckRe
 	if log.DebugEnabled() {
 		log.Debuga("Preprocess Check returned with: ", status.String(out))
 		log.Debug("Dispatching to main adapters after running processors")
-		log.Debuga("Attribute Bag: \n", mutableBag.DebugString())
+		log.Debuga("Attribute Bag: \n", mutableBag.String())
```

Comment:
I think the idea would be to remove the function call entirely, so it doesn't get called unless debug is on...